workspaces/cli-shared/src/captures/avro/file-system/capture-saver.ts

```diff
@@ -21,6 +21,14 @@ export class CaptureSaver implements ICaptureSaver {
     maxSize: 20,
     maxTime: 500,
   });
+
+  private throttler: Bottleneck = new Bottleneck({
```

Comment:
in this usage it's not serving as a throttler, just as a way to track pending batches, so it can be named better